### PR TITLE
Add saltenv kwarg

### PR DESCRIPTION
**Pull request by whiteinge**
Refs #201.

How do you feel about reusing vars and triple fallback redundancy?  ^_^

### CODE REVIEW

File: _modules/hubble.py [hunk 1]
```diff
@@ -404,7 +404,7 @@ def sync():
     '''
     log.debug('syncing nova modules')
     nova_dir = __salt__['config.get']('hubblestack.nova.dir', 'salt://hubblestack_nova')
-    saltenv = __salt__['config.get']('hubblestack.nova.saltenv', 'base')
+    saltenv = saltenv or __salt__['config.get']('hubblestack.nova.saltenv', 'base')
```
Comment (basepi) [inline _modules/hubble.py:420]: Won't this always choose `saltenv`? Because `'base'` is truthy. Probably need to set the default value to `None`.

**Comment by basepi** [comment]
Left a comment inline. `hubble.audit` and `hubble.top` also need to have this argument, and pass through to sync/load when they are called.

**Comment by whiteinge** [comment]
Thanks. And sorry for the half-addition -- we hit non-HubbleStack bugs that prevented us from testing this fully -- I should have held off on the PR.

**Comment by whiteinge** [comment]
Should we do anything with [saltenv in the loader subclass](https://github.com/HubbleStack/Nova/blob/dc9b51268b0b37017bbcfd9f68923ef173238532/_modules/hubble.py#L527) when calling `_hubble_dir()`?

**Comment by basepi** [comment]
Yes, you're right, since I use configuration with all of those values we'll need to pass them through to the loader instantiation, and from there into hubble_dir().

**Comment by whiteinge** [comment]
Added to the loader but this still isn't working. Closing this for now. I have dedicated time next sprint to investigate so I'll reopen this PR at that time.

**Comment by basepi** [comment]
Yeah, you need to modify hubble_dir to actually take that pass-through. I look forward to when you have time to get back to this, should be good.

**Pull request closed without merging**